SMARTCITIES/mobility/MobilityRegion/schema.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
I do not know if it is necessary to have a data model for this. Shouldn't be the limits of the region (perimeter) instead of a Point.
Additionally, where is this data model being used?
The ADOPTERS.yaml should be filled. (see [contribution manual](https://bit.ly/contribution_manual) for it)